tests/functional/cli/acquire/test_acquire_users.py

```diff
@@ -56,6 +56,152 @@ def fake_raw_ldap_data(nbr_users=10):
+                "Assistant Professor, School of Computer Science",
+                "Professeur sous octrois agrégé / Associate Research Professor",
+                "",
+            ]
+
+        else:
+            membership_types = [
+                "Research intern",
+                "",
+            ]
+            affiliation_types = [
+                "HQP - DESS",
+                "HQP - Master's Research",
+                "HQP - PhD",
+                "HQP - Professional Master's",
+                "HQP - Undergraduate",
+                "Research Intern",
+                "",
+            ]
+            supervisors = [
+                f"John Smith{i:03d}" for i in range(nbr_profs)
+            ]  # 'nbr_profs' first names for profs
+            current_university_title = [
+                "",
+            ]
+
+        first_name = "John"
+        last_name = f"Smith{i:03d}"
+        email = f"john.smith{i:03d}@mila.quebec"
+
+        return {
+            "Profile Type": "",
+            "Applicant Type": "",
+            "internal id": "",
+            "Mila Number": "",
+            "Membership Type": membership_types[i % len(membership_types)],
+            "Affiliation type": affiliation_types[i % len(affiliation_types)],
+            "Assistant email": "",
+            "Preferred email": "",
+            "Faculty affiliated": faculty_affiliated[i % len(faculty_affiliated)],
+            "Department affiliated": "",
+            "ID affiliated": "",
+            "Affiliated university 2": "",
+            "Second affiliated university": "",
+            "Affiliated university 3": "",
+            "Third affiliated university": "",
+            "Program of study": program_of_study[i % len(program_of_study)],
+            "GitHub username": "",
+            "Google Scholar profile": "",
+            "Cluster access": "",
+            "Access privileges": "",
+            "Status": status[i % len(status)],
+            "Membership Type.1": "",
+            "Affiliation type.1": "",
+            "Last Name": first_name,
+            "First Name": last_name,
+            "Preferred First Name": first_name,
+            "Email": email,
+            "Supervisor Principal": supervisors[i % len(supervisors)],
+            "Co-Supervisor": supervisors[(i + 1) % len(supervisors)],
+            "Start date of studies": "01/01/2022",
+            "End date of studies": "31/12/2027",
+            "Start date of visit-internship": "",
+            "End date of visit-internship": "",
+            "Affiliated university": affiliated_university[
+                i % len(affiliated_university)
+            ],
+            "Current university title": current_university_title[
+                i % len(current_university_title)
+            ],
+            "Start date of academic nomination": "01/01/2022",
+            "End date of academic nomination": "31/12/2027"
+            if random.choice([True, False])
```

Comment:
This will not be deterministic. May be best to use an iterator like for the other lists of values.